app/services/validation_beta_service.rb

```diff
@@ -25,122 +25,6 @@ def remind_sits_to_add_participants
     end
   end
 
-  # This timestamp was generated by finding the last user with request_for_details_sent_at: nil,
-  # after automation this request for details is sent automatically
-  AUTOMATION_LAUNCH_TIME = Time.zone.parse("2021-09-17 09:45:33.284979000 +0000").freeze
-  VALIDATION_CHASER_BATCH_LIMIT = 5000
-
```

Comment:
We are deleting all unit tests for this servise, but not all the contents of this service. Does that sound right? :P

Reply:
Not all of them, but quite a lot of them - enough for github to truncate the diff for the file. You can click on "Load the diff" - there are about 300 lines of tests deleted for the deleted methods, the remaining tests are still there.

Reply:
Ahh, my bad. Alright then :)